Framework/CurveFitting/src/Algorithms/RefinePowderInstrumentParameters3.cpp

```diff
@@ -1198,10 +1198,10 @@ void RefinePowderInstrumentParameters3::setFunctionParameterFitSetups(const IFun
  * exacly same as
  * source;
  */
-void duplicateParameters(map<string, Parameter> source, map<string, Parameter> &target) {
+void duplicateParameters(const map<string, Parameter> source, map<string, Parameter> &target) {
```

Comment:
```suggestion
void duplicateParameters(const map<string, Parameter> &source, map<string, Parameter> &target) {
```
That'll sort the cppcheck warning, (as well as sorting the header file)